scripts/promote-beta-experimental-traffic.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Promotes a Beta/Experimental traffic release.
+ */
+
+import minimist, {ParsedArgs} from 'minimist';
+import {createVersionsUpdatePullRequest, runPromoteJob} from './promote-job';
+
+interface Args extends ParsedArgs {
+  amp_version?: string;
+}
+
+const jobName = 'promote-beta-experimental-traffic.ts';
+const {amp_version: AMP_VERSION}: Args = minimist(process.argv.slice(2));
+
+async function rtvExists(rtv: string): Promise<string | null> {
+  const response = await fetch(`http://cdn.ampproject.org/rtv/${rtv}/v0.js`);
```

Comment:
Shouldn't this check the experiments config as the source of truth, not whether a certain RTV is live?

Reply:
The experiments config file is in another repo, and checking out all of amphtml just for that file is an overkill imo

We _could_ fetch that config for this specific AMP version (either as a `fetch` or using Octokit), this assumes the version was tagged correctly - did we ever have a problem with that? If not, then I think using Octokit would be the way to go

Reply:
Why not fetch a single file from raw.githubusercontent.com?

Reply:
No particular reason, but since we're already "inside" github with a token I figured that's the "correct" way to do this. Just finished making this change: 44118e2